### ISSUE TITLE
Developer Guide: How to get started

### ISSUE THREAD

**Issue by jensbuehl** [LENpolygon/Build-Order-Tool-AoE4-#38]
In order to support here and there and try out stuff:
Since I (and maybe some other interested developers) do not know how to setup the dev environment properly, get it running and prepare for local testing.

Whoever is "owner" or kicked this project off: Would a tiny bullet point list in the readme.md be possible?
Would be highly appreciated.

**Comment by LENpolygon** [comment]
Hi Jenzb!
Thanks for this request and your other posts & interest in this project!

I'm not much of a developer/programmer myself. So it's pretty low tech:

- Download the entire Github in a .zip file
- Unzip to your folder of choice
- Open index.html and see if the website loads (the icons will be replaced to prevent spreading around copyrighted images.)
- Edit files with your favorite code editing tool. I like using Visual Studio Code

Html is in main folder and Javascript / CSS are in seperate folders. I've tried splitting up JS in sensible blocks.
It's just Vanilla JS. Nothing special.

**Comment by jenzb** [comment]
Thx. :)
Maybe I just start a new conversation in the discussions area for specific details, if needed.

**Comment by JarrettSpiker** [comment]
A quick section on how to set up a firebase config would also be great! The app wont work locally until you have one

**Comment by davidgamero** [comment]
Good idea- if @LENpolygon wants to confirm it's okay to use the local firebase config or make a dev firebase config I'm happy to write it up!
